documentation/Database-Schema-Application.md

```diff
@@ -0,0 +1,56 @@
+# Database Schema Application
+
+![Database Schema](images/database-schema-application.png)
+
+## Tables description: Application
+
+### Application instance
+The application has all the visualisation and logic defined in the <b>application template</b> [(described in Database Schema Template)](Database-Schema-Template.md). When the Applicant clicks to apply for a application type (aka template) a new instance of one application is created for them. The Applicant user can make changes until submitting the application to be reviewed.
+
+* <b>application</b>
+Representation of the application instance. All nested elements are accessible via joined tables and can be created or queried in the same call using the GraphQL engine.
+The `template_id` links to the template used to display all sections and elements (questions) to the user.
+The `user_id` links to the Applicant user that created this application.
+The `serial` is a number used to display the application number to users. 
+The `name` will be deduced by the template name + user + company names.
+The `is_active` is set to `'true'` for all applications that are still in the review process. 
+The `is_active` is set to `'false'` for applications that have finished the review process, have expire or were withdrawn by the Applicant. 
+The `trigger` is updated everytime the application has changes done by users or a scheduler. The trigger would be one of the options: `'onApplicationCreate'`, `'onApplicationSubmit'`, `'onApplicationSave'`, `'onApplicationWithdrawn'`, `'onReviewStart'`, `'onReviewEditComment'`, `'onReviewSave'`, `'onReviewAssign'`, `'onApprovalSubmit'`, `'onScheduleTime'`.
```

Comment:
Agghh -- not more places to maintain the list of Triggers 🤣

See my comment on #24 -- maybe best to just say "See here for detailed list of available triggers" with a link to the Triggers & Actions page.